README.md

```diff
@@ -9,19 +9,30 @@
 
 ## Quick Start
 
-### Python
+To use the GX Agent, you will need to have a Great Expectations Cloud account. If you don't have one, you can sign up for free at [https://app.greatexpectations.io](https://app.greatexpectations.io).
```

Comment:
```suggestion
To use the GX Agent, you will need to have a free [Great Expectations Cloud](https://app.greatexpectations.io) account.
```

Reply:
My suggestion is: To use the GX Agent, you will need to have a Great Expectations Cloud account. You can sign up for free at [https://app.greatexpectations.io](https://app.greatexpectations.io).

Reply:
I was just going for more concise, but I dont feel strongly